lib/rules/sotd/pp.js

```diff
@@ -81,6 +84,22 @@ function findPP (candidates, sr) {
     return pp;
 }
 
+function fetchGroupName(deliverIds) {
+    return new PowerPromise(function (resolve, reject) {
+        w3capi.apiKey = process.env.W3C_API_KEY;
+        w3capi.group(deliverIds[0]).fetch((err, data) => {
+            if (err) {
+                reject();
+            } else {
+                resolve(data.name);
+            }
+        });
+    });
+}
+
+    // deliverIds.some(deliverId => {
```

Comment:
Delete?

Reply:
Hi @nschonni , thanks for the review. This PR is not finished, just push to view diff and discuss. This `deliverIds` is an array and need iteration, I'll continue working on it today.

Reply:
Sorry about that, I missed the Draft status